src/main/java/org/jitsi/videobridge/SctpConnection.java

```diff
@@ -1080,6 +1080,9 @@ public void run()
         {
             do
             {
+                if (sctpSocket == null)
+                    break;
+
```

Comment:
Could I please ask you to write a comment explaining why the check's there? I got the reason from your PR comment but I don't think that's obvious (to me).